content/en/tracing/send_traces/_index.md

```diff
@@ -23,7 +23,9 @@ further_reading:
   text: "ECS Fargate APM setup"
 ---
 
-To use APM, start by sending your [traces][1] to Datadog, and then [configure your environment](#configure-your-environment). You can send traces to Datadog in multiple different ways depending on your system setup: including using the [Datadog Agent locally](#datadog-agent), [on containers](#containers), and [several other ways](#additional-environments). For the full overview of all of the steps to set up APM, see the [APM overview][2].
+Configuring your application to send [traces][1] to Datadog involves two main steps: Configuring the Datadog Agent and [Instrumenting your application][14]. 
+
+Depending on your environment, there are different ways to configure the Datadog Agent. Instructions are provided here for [installing the Datadog Agent locally](#datadog-agent), [on containers](#containers), and [several other ways](#additional-environments).
```

Comment:
```suggestion
Depending on your environment, there are different ways to configure the Datadog Agent. See the instructions for installing the Datadog Agent [locally](#datadog-agent), [on containers](#containers), and [several other ways](#additional-environments).
```

Reply:
Also, this kinda just skips over serverless. Want to list it here too?

Reply:
Yeah Serverless we have the separate page for serverless that we'd want to direct people to if they are here.  What's a natural way to do that?